server/models/Profile.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+
+const ProfileSchema = new mongoose.Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+    },
+    gender: {
+      type: String,
+    },
+    birthDate: {
+      type: Date,
+    },
+    email: {
+      type: String,
+      required: true,
+    },
+    address: {
+      type: String,
+    },
+    availabiltity: {
```

Comment:
Please give me an example of how this design for availability will work? :)

Reply:
I am trying to set an array that contains a list of objects that each would key for dates and days which the user will be available:

Reply:
Example
["Sun 2am-4pm", "Mon 4am-6am"]

Reply:
Thanks for the clarification, the UI will have to be designed accordingly 👍🏽